sdk/ml/azure-ai-ml/CHANGELOG.md

```diff
@@ -1,16 +1,20 @@
 # Release History
 
-## 1.17.0 (unreleased)
+## 1.18.0 (unreleased)
 
 ### Features Added
 
 ### Bugs Fixed
-- InputTypes exported in constants module
 
 ### Breaking Changes
 
 ### Other Changes
 
+## 1.17.0 (2024-06-13)
+
+### Bugs Fixed
+- InputTypes exported in constants module
+
```

Comment:
I'm pretty sure this InputTypes PR was merged after the release cut was taken, so it should be in 1.18 i'm pretty sure